src/languageclient/reloadSchemas.ts

```diff
@@ -11,8 +12,10 @@ export async function reloadSchemas(): Promise<void> {
         ext.outputChannel.appendLine("Reloading schemas");
 
         await ext.languageServerClient.sendRequest('schema/reloadSchemas', {});
+
+        await window.showInformationMessage("Azure schema cache has been cleared. Please restart Visual Studio Code to reload schemas.");
```

Comment:
Normally I avoid awaiting `showInformationMessage` since I never care about the user's response and we don't get telemetry until they dismiss the notification (which might never happen)

Reply:
Makes sense, thx